modules/functions.js

```diff
@@ -57,11 +57,14 @@ module.exports = (client) => {
   and stringifies objects!
   This is mostly only used by the Eval and Exec commands.
   */
+  
+  var util = require("util");
```

Comment:
We do not use `var`. Use `const` instead. [prefer const](https://eslint.org/docs/rules/prefer-const) rule in ESLint.